src/index.tsx

```diff
@@ -5,6 +5,11 @@ import App from "./App";
 import { register } from "./serviceWorkerRegistration";
 import reportWebVitals from "./reportWebVitals";
 
+const meta = document.createElement('meta');
+meta.name = 'viewport';
+meta.content = 'width=device-width, initial-scale=1, maximum-scale=1';
+document.getElementsByTagName('head')[0].appendChild(meta);
+
```

Comment:
we already have a meta tag in the index.html file, with pretty much the same content. You can add the maximum-scale=1 there and remove these lines here